Sources/BufferOutputFacility.swift

```diff
@@ -11,32 +11,32 @@ import Foundation
 @objc(RDTBufferOutputFacility) public final class BufferOutputFacility: NSObject, OutputFacility {
 
 	/// A buffer of request representations.
-	public private(set) var requestRepresentations: [RequestRepresentation] = []
+	public fileprivate(set) var requestRepresentations: [RequestRepresentation] = []
```

Comment:
Why do you prefer `private` over `fileprivate`? I love to use extensions in ways [Natasha The Robot](https://www.natashatherobot.com/using-swift-extensions/) described. In this case you need `fileprivate`